main.nf

```diff
@@ -259,11 +266,11 @@ if(!params.bwa_meth_index && params.fasta && params.aligner == 'bwameth'){
  */
 if(!params.fasta_index && params.fasta && params.aligner == 'bwameth'){
     process makeFastaIndex {
-        tag fasta
+        tag "$fasta"
         publishDir path: "${params.outdir}/reference_genome", saveAs: { params.saveReference ? it : null }, mode: 'copy'
 
         input:
-        file fasta
+        file fasta from fasta_2
```

Comment:
Can you please use names that are more explicit? 

`ch_from_blabla_to_blabla` is something that helps you (and me, others) to keep things traceable :-)